openupgrade_scripts/apriori.py

```diff
@@ -43,7 +44,6 @@
     "l10n_es_edi_facturae_adm_centers": "l10n_es_edi_facturae",
     "l10n_es_edi_facturae_invoice_period": "l10n_es_edi_facturae",
     "l10n_es_edi_tbai_multi_refund": "l10n_es_edi_tbai",
-    "l10n_fr_fec": "l10n_fr",
```

Comment:
"l10n_fr_fec": "l10n_fr_account".

Reply:
`l10n_fr_account` is a split of `l10n_fr` in which `l10n_fr_fec` was merged into. Is there a problem?

Reply:
That conceptualization doesn't matter at this level. The fact is that `l10n_fr_account` didn't exist previously, and `l10n_fr_fec` did, but not now, so it's a clear rename. The rest of the operations (renames and so on) should be done on scripts, but not on apriori.py.

There's a risk (although minimal), that putting it in `merged_modules`, some information is lost (as there are deletes), or that some important crashes are ignored.

And also talking about that, if this is a split as you say, you should force its installation in `l10n_fr` scripts.